framework/client/releasing.rst

```diff
@@ -0,0 +1,80 @@
+Releasing
+=================
+
+New Feature
+-----------
+New features will be developed in separate feature/pull-request branches. As soon as a feature is ready, a pull request is opened to rebase the feature into the master branch.
+After a successful rebase the feature branch can be deleted. Only self-contained features should be rebased to master to keep master stable and packages always ready to be published.
```

Comment:
maybe "should be deleted" instead of "can be deleted"? IMO there's no use in keeping merged feature branches, they just clutter everything :)